libs/domain/search/facet-rating/facet-rating.def.ts

```diff
@@ -1,7 +1,7 @@
 import { componentDef } from '@spryker-oryx/utilities';
 
 export const searchRatingFacetComponent = componentDef({
-  name: 'oryx-search-rating-facet',
```

Comment:
We can't rename existing component.

Reply:
But it is a new component. Maybe you mean the similar change in `color facet` component?

Reply:
Exactly.